pkg/manager/manager_table.go

```diff
@@ -36,6 +39,19 @@ func (sm *Manager) startTableMode(id string) error {
 		}()
 	}
 
+	egressCleanEnv := os.Getenv("EGRESS_CLEAN")
```

Comment:
This is now `vip.ClearIPTables(sm.config.EgressWithNftables, sm.config.ServiceNamespace, iptables.ProtocolIPv4)` in places like `manager_arp.go`, should this be replaced here too?

Reply:
I don't quite understand. :smile:  `vip.ClearIPTables` function is used here as well as in other places I believe.